packages/api/routes/apikeys/key/createKeyMiddleware.js

```diff
@@ -0,0 +1,17 @@
+const db_apikey = require("../../../lib/db.apikey");
+
+// Return a function for creating an API key for a given user.
+module.exports = function createKeyMiddleware() {
+  return async (req, res, next) => {
+    const apikey = await db_apikey.attach();
+    const user = req.body.user ? req.body.user.trim() : req.body.user;
+    const doc = user
+      ? await apikey.createKey(user)
+      : {
+          error: "Invalid Parameter",
+          message: "Username Missing: Username cannot be empty. It is needed to create and assign a new API Key."
```

Comment:
I suggest changing this to "user id".  The value for `user` can be any string, but it's meant to be the user's UUID.
```suggestion
          message: "user id missing: user id cannot be empty. It is needed to create and assign a new API Key."
```